client/src/index.tsx

```diff
@@ -44,7 +44,18 @@ const link = split(
 
 export const client = new ApolloClient({
     link,
-    cache: new InMemoryCache(),
+    cache: new InMemoryCache({
```

Comment:
what does this do?

Reply:
Apollo normalizes the cache automatically.  That means if your GraphQL API returns a Book with an Author, in the cache (like relational database tables) you have Books and Authors stored separately.  The catch is that Book and Author must have an `id` or `_id` property, or else they won't be updated after a client-side mutation is made.  So the one function here tells Apollo "my id property is actually named this" so the cache updates after mutations or subscription events work.